determined-ai/determined
# fix: use post for searchRuns/use accurate multitrial detection

## Description
this fixes a few issues with the multitrial check for run actions

* when the selection grew too large, the resulting url triggered a 431 error. we're switching the method of the search runs endpoint to post in order to get around this.
* the multitrial check used the wrong method to check if a run was part of a search
* the warning shown if the multitrial check returned positive had incorrect copy



## Test Plan
* navigate to the runs table
* select all runs in the current page
* select the move action
* the check should succeed and the notice should show that multitrial runs will be moved along with other runs in their search



## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
